benchmarks/graphics_pipeline/main.cpp

```diff
@@ -789,80 +252,77 @@ void ProjApp::Setup()
-            /* binding= */ 0,
-            /* type= */ grfx::DESCRIPTOR_TYPE_UNIFORM_BUFFER,
-            /* array_count= */ 1,
-            /* shader_visibility= */ grfx::SHADER_STAGE_ALL_GRAPHICS});
-
-        // Albedo
-        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{
-            /* binding= */ 1,
-            /* type= */ grfx::DESCRIPTOR_TYPE_SAMPLED_IMAGE,
-            /* array_count= */ 1,
-            /* shader_visibility= */ grfx::SHADER_STAGE_PS});
-        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{
-            /* binding= */ 2,
-            /* type= */ grfx::DESCRIPTOR_TYPE_SAMPLER,
-            /* array_count= */ 1,
-            /* shader_visibility= */ grfx::SHADER_STAGE_PS});
-
-        // Normal
-        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{
-            /* binding= */ 3,
-            /* type= */ grfx::DESCRIPTOR_TYPE_SAMPLED_IMAGE,
-            /* array_count= */ 1,
-            /* shader_visibility= */ grfx::SHADER_STAGE_PS});
-        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{
-            /* binding= */ 4,
-            /* type= */ grfx::DESCRIPTOR_TYPE_SAMPLER,
-            /* array_count= */ 1,
-            /* shader_visibility= */ grfx::SHADER_STAGE_PS});
-
-        // Metallic/Roughness
-        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{
-            /* binding= */ 5,
-            /* type= */ grfx::DESCRIPTOR_TYPE_SAMPLED_IMAGE,
-            /* array_count= */ 1,
-            /* shader_visibility= */ grfx::SHADER_STAGE_PS});
-        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{
-            /* binding= */ 6,
-            /* type= */ grfx::DESCRIPTOR_TYPE_SAMPLER,
-            /* array_count= */ 1,
-            /* shader_visibility= */ grfx::SHADER_STAGE_PS});
-
-        PPX_CHECKED_CALL(GetDevice()->CreateDescriptorSetLayout(&layoutCreateInfo, &mSetLayout));
+        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{0, grfx::DESCRIPTOR_TYPE_UNIFORM_BUFFER, 1, grfx::SHADER_STAGE_ALL_GRAPHICS});
+        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{1, grfx::DESCRIPTOR_TYPE_SAMPLED_IMAGE, 1, grfx::SHADER_STAGE_ALL_GRAPHICS});
+        layoutCreateInfo.bindings.push_back(grfx::DescriptorBinding{2, grfx::DESCRIPTOR_TYPE_SAMPLER, 1, grfx::SHADER_STAGE_ALL_GRAPHICS});
+        PPX_CHECKED_CALL(GetDevice()->CreateDescriptorSetLayout(&layoutCreateInfo, &mDescriptorSetLayout));
     }
 
-    for (size_t i = 0; i < kAvailableScenesFilePath.size(); i++) {
-        LoadScene(
-            kAvailableScenesFilePath[i],
-            GetDevice(),
-            GetSwapchain(),
-            GetGraphicsQueue(),
-            mDescriptorPool,
-            &mTextureCache,
-            &mScenes[i].objects,
-            &mScenes[i].primitives,
-            &mScenes[i].materials);
+    // Entities
+    {
+        TriMesh mesh = TriMesh::CreateCube(float3(8, 8, 8));
```

Comment:
not really a big deal, but you have a scale value later, i guess you can put 1,1,1 here, and only use the scale to control the size.